app/models/manageiq/providers/ansible_tower/automation_manager/refresh_worker/parser.rb

```diff
@@ -8,10 +8,14 @@ def initialize(ems, target, inventory)
   end
 
   def parse
+    parse_configuration_script_sources
```

Comment:
hm, so this starts to break the pattern

so I would rather move to init all ICs first, parse Collector data, return array of ICs